test/e2e/network_test.go

```diff
@@ -233,6 +231,12 @@ func testNorthSouthNetworking(t *testing.T) {
 	testCtx, err := NewTestContext()
 	require.NoError(t, err)
 
+	// Ignore the application ingress load balancer test for None and vSphere platforms as it has to be created manually
+	// https://docs.openshift.com/container-platform/4.5/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer.html#nw-using-load-balancer-getting-traffic_configuring-ingress-cluster-traffic-load-balancer
+	if testCtx.CloudProvider.GetType() == config.VSpherePlatformType ||
+		testCtx.CloudProvider.GetType() == config.NonePlatformType {
+		t.Skipf("NorthSouthNetworking test is disabled for platform %s", testCtx.CloudProvider.GetType())
```

Comment:
Can you edit the commit message to say you are using t.Skipf?

Reply:
Good catch!

Fixed.